tools/clang/unittests/HLSL/DxilContainerTest.cpp

```diff
@@ -442,6 +443,25 @@ class DxilContainerTest : public ::testing::Test {
   }
 #endif // _WIN32 - DXBC Unsupported
 
+  void CheckResult(LPCWSTR operation, IDxcOperationResult *pResult) {
+    HRESULT hrStatus = S_OK;
+    VERIFY_SUCCEEDED(pResult->GetStatus(&hrStatus));
+    if (FAILED(hrStatus)) {
+      LogErrorFmt(L"%s failed with hr %d.", operation);
```

Comment:
Shouldn't there be another object here? I don't see what %d is formatting, but I see operation goes into %s.